pkg/cloud/alibaba/assets/cloud-controller-manager-daemonset.yaml

```diff
@@ -0,0 +1,96 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cloud-config
+  namespace: kube-system
+data:
+  cloud-config.conf: |-
+    {
+        "Global": {
+            "routeTableIDs": ""
+        }
+    }
+---
+apiVersion: apps/v1
+kind: DaemonSet
```

Comment:
This should probably be a deployment to match the other platforms in this project. Please copy one of the other clouds and adjust as necessary for Ali Cloud